itou/api/employee_record_api/viewsets.py

```diff
@@ -50,6 +52,39 @@ class AbstractEmployeeRecordViewSet(viewsets.ReadOnlyModelViewSet):
     permission_classes = [EmployeeRecordAPIPermission]
 
 
+def _annotate_convert_created_at(queryset):
+    # Add a new `creation_date` field (cast of `created_at` to a date)
+    return queryset.annotate(creation_date=Cast("created_at", output_field=DateField()))
+
+
+def _filter_by_query_params(request, queryset):
+    """
+    Register query parameters result filtering:
+    - only using employee record `status` is available for now.
+    - `status` query param can be an array of value.
+    """
+    result = queryset
+    params = request.query_params
+
+    if not params:
+        return result.processed().order_by("-created_at")
+
+    # Query params are chainable
+
+    if status := params.getlist("status", ""):
```

Comment:
_nit_ : Il me semble que `getlist()` renvois par défaut une valeur _falsy_, et c'est bizarre d'avoir une chaîne vide alors qu'on parle de liste :)
```suggestion
    if status := params.getlist("status"):
```

Reply:
oui !